readme.md

```diff
@@ -221,6 +221,10 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 ---
 
 ## Changelog
+
+### 4.1.1
```

Comment:
No need to change here, but if we merge the dependency upgrades as part of our next release I'd prefer we go with a minor version bump at a minimum.